test/clubbing_top_level.txt

```diff
@@ -0,0 +1 @@
+<table border="1"><tr><th>aaa</th><th>ccc</th></tr><tr><td>1</td><td>2</td></tr><tr><td>bbb</td><td>2</td></tr></table>
```

Comment:
Aight, so here's my update:

1. I just added a clubbing switch to convert. 

2. The behaviour where a list with a single dict entry becomes a clubbed table is actually not my change but that of @lekic, which came in with 1.0.3. All I did was extend the same behaviour of embedded lists to top level JSON lists. I agree with @Lekic's idea however. Here's my reasoning: 

Consider the following json:

    [{"a":1}, {"a":2}]

Here we agree it should be clubbed, right? So we get a table. Now take away one of the dictionaries from the list. Wait, what happened, why does my output now look completely different? First I had a table, now I have a list with one entry, itself being a table? 

Would you agree that for consistency reasons, if clubbing is activated, a list of one elementsshould behave the same as a list of two elements, iff they have the same properties (thus can be clubbed)? Otherwise I'd find it a surprising result. 

So now, if people don't want this they can just convert with the `clubbing = False` option. In that case, again, it behaves the same if the list has one or multiple elements.